<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgShowDeptch</class>
 <widget class="QDialog" name="DlgShowDeptch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>147</width>
    <height>127</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>147</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Значение глубины</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="EditX"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="EditY"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Значение:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="EditBrigth"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Канал:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="ComboCh"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
